app/src/main/java/com/fcaputo/parkingapp/mvp/view/ReleaseView.kt

```diff
@@ -0,0 +1,79 @@
+package com.fcaputo.parkingapp.mvp.view
+
+import android.app.Activity
+import androidx.annotation.StringRes
+import androidx.appcompat.app.AppCompatActivity
+import com.fcaputo.parkingapp.R
+import com.fcaputo.parkingapp.databinding.ActivityReleaseBinding
+import com.fcaputo.parkingapp.mvp.contract.ReleaseContract
+import com.fcaputo.parkingapp.mvp.view.base.ActivityView
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+
+class ReleaseView(activity: Activity) : ActivityView(activity), ReleaseContract.View {
+    private var binding: ActivityReleaseBinding = ActivityReleaseBinding.inflate(activity.layoutInflater)
+
+    init {
+        activity.setContentView(binding.root)
+    }
+
+    override fun onReleaseButton(onClick: () -> Unit) {
+        binding.btnRelease.setOnClickListener { onClick() }
+    }
+
+    override fun getEditTextsContents(): List<String> {
+        return listOf(binding.etParkingNumber.text.toString(),
+            binding.etSecurityCode.text.toString()
+        )
+    }
+
+    override fun showIncompleteFieldError() {
+        showErrorMessage(R.string.error_incomplete_field_title, R.string.error_incomplete_field_msg)
+    }
+
+    override fun showConfirmationDialog(onConfirmed: () -> Unit) {
+        MaterialAlertDialogBuilder(activity as AppCompatActivity)
```

Comment:
I recommend creating an AlertDialogUtils or AlertDialogFactory class, so you can abstract the creation and behavior. Doing so, from the views you will only call AlertDialogFactory.showDialog(...)